File: app/src/main/java/com/chad/baserecyclerviewadapterhelper/entity/Status.java
package com.chad.baserecyclerviewadapterhelper.entity;

public class Status {
    private boolean isRetweet;
    private String text;
    private String userName;
    private String userAvatar;
    private String createdAt;

    public boolean isRetweet() {
        return isRetweet;
    }

    public void setRetweet(boolean retweet) {
        isRetweet = retweet;
    }

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }

    public String getUserName() {
        return userName;
    }

    public void setUserName(String userName) {
        this.userName = userName;
    }

    public String getUserAvatar() {
        return userAvatar;
    }

    public void setUserAvatar(String userAvatar) {
        this.userAvatar = userAvatar;
    }

    public String getCreatedAt() {
        return createdAt;
    }

    public void setCreatedAt(String createdAt) {
        this.createdAt = createdAt;
    }

    @Override
    public String toString() {
        return "Status{" +
                "isRetweet=" + isRetweet +
                ", text='" + text + '\'' +
                ", userName='" + userName + '\'' +
                ", userAvatar='" + userAvatar + '\'' +
                ", createdAt='" + createdAt + '\'' +
                '}';
    }
}

File: kotlinapp/src/main/java/com/allen/kotlinapp/entity/Status.kt
package com.allen.kotlinapp.entity

/**
 * 文 件 名: Status
 * 创 建 人: Allen
 * 创建日期: 2017/6/13 14:02
 * 修改时间：
 * 修改备注：
 */

class Status {
    var isRetweet: Boolean = false
    var text: String? = null
    var userName: String? = null
    var userAvatar: String? = null
    var createdAt: String? = null

    override fun toString(): String {
        return "Status{" +
                "isRetweet=" + isRetweet +
                ", text='" + text + '\'' +
                ", userName='" + userName + '\'' +
                ", userAvatar='" + userAvatar + '\'' +
                ", createdAt='" + createdAt + '\'' +
                '}'
    }
}
